### PR TITLE
Feature/align starting chain dropdown with connected chain

### PR DESCRIPTION
**Pull request by michalsidzej**

### CODE REVIEW

File: src/pages/Aggregator/Aggregator.hooks.ts [hunk 1]
```diff
@@ -95,6 +100,35 @@ export const useAggregatorDropdowns = (
...
+) => {
+  const connectedChain = useSelector(chainIdSelector);
+  const provider = useSelector(providerSelector);
+
+  const wrongChainConnectedError = startingChain !== connectedChain;
+
+  useEffect(() => {
+    if (startingChain && wrongChainConnectedError) {
+      provider?.send('wallet_switchEthereumChain', [
+        { chainId: `0x${startingChain.toString(16)}` },
```
Comment (meho99) [inline src/pages/Aggregator/Aggregator.hooks.ts:115]: I think you can use hexlify here:

import { utils } from 'ethers';
chainId: utils.hexlify(startingChain)

**Review by meho99 — COMMENTED** [review]
Nice
Remember to add some tests for the `useConnectedChain` when we will have more free time in the project

File: src/pages/Aggregator/Aggregator.hooks.ts (hunk 1 above)
Comment (michalsidzej) [reply src/pages/Aggregator/Aggregator.hooks.ts:115]: Tried it, but theres a problem with `ethereum mainnet`
Zrzut ekranu z 2022-03-22 14-12-29

Comment (michalsidzej) [reply src/pages/Aggregator/Aggregator.hooks.ts:115]: > Nice! Remember to add some tests for the `useConnectedChain` when we will have more free time in the project

Added as todo on Clickup

**Review by meho99 — APPROVED** [review]

**Pull request merged**